core-services/prow/02_config/openshift/openstack-test/_prowconfig.yaml

```diff
@@ -8,6 +8,7 @@ tide:
     - release-4.12
     - release-4.13
     - release-4.14
+    - release-4.15
```

Comment:
There too, we should add the release-4.16 branch

Reply:
I don't think so. We will only merge to release-4.16 once it's no longer synced to the main branch

Reply:
You're right.